neon_utils/skill_override_functions.py

```diff
@@ -351,3 +350,146 @@ def to_system_time(dt):
         return dt.astimezone(tz)
     else:
         return dt.replace(tzinfo=gettz("UTC")).astimezone(tz)
+
+
+def speak(utterance, expect_response=False, message=None, private=False, speaker=None, wait=False, meta=None):
```

Comment:
This was already handled in a separate PR; please rebase changes

https://github.com/NeonGeckoCom/neon-skill-utils/blob/5d7d0f61ac595f2b94b773b59cc2460749f14fb3/neon_utils/skill_override_functions.py#L123